recipe/patch_yaml/ome-zarr.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../patch_yaml_model.json
+# from this bit of code
+
+# zarr 2.18.x dropped support for Python 3.10 in a patch release with
+# https://github.com/zarr-developers/zarr-python/pull/2344 following
+# https://scientific-python.org/specs/spec-0000/
+
+# PRs https://github.com/conda-forge/ome-zarr-feedstock/pull/22
+# and https://github.com/conda-forge/ome-zarr-feedstock/pull/23
+# dealt with the immediate build issues.
+if:
+  name: ome-zarr
+  version_lt: "0.10.3"
+  timestamp_lt: 1736594995
+  # Saturday, January 11, 2025 12:29:55 PM GMT+01:00
+then:
+  - tighten_depends:
+      name: zarr
+      upper_bound: 3.0.0
```

Comment:
This doesn't seem to have the intended effect.

Reply:
Try running: 
```
python show_diff.py --subdirs noarch
```

locally

Reply:
Looking like even less ...

```
(conda-patches) /opt/conda-forge-repodata-patches-feedstock/recipe $python show_diff.py --subdirs noarch
================================================================================
================================================================================
noarch
```

Reply:
right, so now you can debug why your patch isn't working.

Reply:
the bug was that you needed `000` at the end of your timestamp. we use `milliseconds` not `seconds`.